daemon/daemon.go

```diff
@@ -981,3 +984,30 @@ func (daemon *Daemon) newBaseContainer(id string) CommonContainer {
 		root:         daemon.containerRoot(id),
 	}
 }
+
+func setDefaultMtu(config *Config) {
+	// do nothing if the config does not have the default 0 value.
+	if config.Mtu != 0 {
+		return
+	}
+	config.Mtu = defaultNetworkMtu
+	if routeMtu, err := getDefaultRouteMtu(); err == nil {
```

Comment:
I think we should do something with the error when `err != nil`. Right now we're just ignoring it.